package.json

```diff
@@ -248,9 +248,26 @@
       ],
       "explorer/context": [
         {
-            "command": "java.view.package.revealInProjectExplorer",
-            "when": "resourceExtname == .java && java:projectManagerActivated && java:serverMode == Standard",
-            "group": "java:project_10"
+          "command": "java.view.package.revealInProjectExplorer",
+          "when": "resourceFilename in java:supportedBuildFiles && java:projectManagerActivated && java:serverMode == Standard",
+          "group": "navigation@100"
+        },
+        {
+          "command": "java.view.package.revealInProjectExplorer",
+          "when": "resourceExtname == .java && java:projectManagerActivated && java:serverMode == Standard",
```

Comment:
paranthesis in "when" is not supported so here I split them into two part: https://github.com/microsoft/vscode/issues/91473